ember-cli-build.js

```diff
@@ -4,7 +4,9 @@ const EmberApp = require('ember-cli/lib/broccoli/ember-app');
 
 module.exports = function(defaults) {
   let app = new EmberApp(defaults, {
-    // Add options here
+    babel: {
+        sourceMaps: 'inline'
+      }
```

Comment:
We don't want inline source maps for production builds. Use `EmberApp.env()` for deciding whether this value should be set or not.